### PR TITLE
Build script enable windows arm64 target

### PR DESCRIPTION
**Pull request by brgavino**
Enable Windows on arm64 compile target support using the VC cross compile toolchain, some modifications to build_win.py and owt\talk\BUILD.gn to support compilation

### CODE REVIEW

File: scripts/build-win.py [hunk 1]
```diff
@@ -39,6 +38,8 @@
 def gngen(arch, ssl_root, msdk_root, quic_root, scheme, tests):
     gn_args = list(GN_ARGS)
     gn_args.append('target_cpu="%s"' % arch)
+    if arch != 'arm64':
```
Comment (jianjunz) [inline scripts/build-win.py:41]: if arch!='arm64' and scheme=='release', there will be two `is_debug=false`.

File: talk/owt/BUILD.gn [hunk 2]
```diff
@@ -240,22 +247,13 @@ static_library("owt_sdk_base") {
...
-    include_dirs += [
-      "//third_party/libvpx/source/config",
-      "//third_party/libvpx/source/config/$os_category/$cpu_arch_full",
-      "//third_party/libvpx/source/libvpx",
-    ]
+    
   }
   if (is_linux) {
     if (owt_msdk_header_root != "") {
-      include_dirs += [ owt_msdk_header_root ]
```
Comment (jianjunz) [inline talk/owt/BUILD.gn:257]: This is for Linux, keep it?

**Review by jianjunz — COMMENTED** [review]
@taste1981, please take a look as well. Thanks.

File: talk/owt/BUILD.gn [hunk 3]
```diff
@@ -8,21 +8,20 @@ import("//testing/test.gni")
...
-  }
+ 
+ if (current_cpu == "x86") {
+   cpu_arch_full = "ia32"
+ } else if (current_cpu == "x64") {
+   cpu_arch_full = "x64"
+ } else if (current_cpu == "arm") {
+   if (arm_use_neon) {
+     cpu_arch_full = "arm-neon"
+   } else if (is_android) {
```
Comment (taste1981) [inline talk/owt/BUILD.gn:19]: under is_win, there is no possibility for is_android to be defined. Although this is not your issue, can you remove it?

File: scripts/build-win.py (hunk 1 above)
Comment (brgavino) [reply scripts/build-win.py:41]: I don't see the logic for this, the conditional for `if arch != 'arm64':` is to set `is_clang=false`, the next conditional will only set `is_debug=false`

Comment (jianjunz) [reply scripts/build-win.py:41]: You're right. Sorry for my incorrect comment above.

**Review by jianjunz — DISMISSED** [review]

**Comment by taste1981** [comment]
@brgavino basically I'm fine with this. Please resolve conflicts until we can further review it. Thanks!

**Review by jianjunz — APPROVED** [review]

**Review by taste1981 — APPROVED** [review]

**Pull request merged**